### DISCUSSION TITLE
Giro stabilized differential thrust

### DISCUSSION THREAD

**Discussion by MrD-RC** [iNavFlight/inav#7708] [Q&A]
### Discussed in https://github.com/iNavFlight/inav/discussions/7704 https://github.com/iNavFlight/inav/discussions/7691

<div type='discussions-op-text'>

<sup>Originally posted by **Taboaco** December 30, 2021</sup>
Hello guys, one last try here, I need to set a wing with differential thrust on inav, so far so good, but I really need the motors to be giro or accelerometer stabilized to avoid “ Dutch roll” kind of turbulence, just lock my yaw like a quad does, I believe it will benefit everyone that flies those kind of FPV tween motor wings and will make the FPV experience way more enjoyable and less shaky, I really need this due a project I have in a not sweep wing with no vertical stabilizer fins, I don’t know it I mix an yaw or gimbal with motors,… but I was unable to do it so!
Please help!
Thanks</div>

**Comment by MrD-RC** [comment]
Issue also created from #7700

**Comment by DzikuVx** [comment]
<img width="1214" alt="image" src="https://user-images.githubusercontent.com/966811/147815192-c6f8a058-19d7-4295-9656-ae75f90f22a4.png">

Differential thrust is added in the mixer table in YAW column. There are even ready made mixer presets with differential thrust to start with

**Comment by MrD-RC** [comment]
@Taboaco Paweł and I have been chatting. Differential thrust is stabilised. If you are not getting enough effect, you will need to tune the differential thrust settings. 

This could be increasing the figure in the yaw columns of the motors mixer. Or the PIDFF and rates for yaw need to be setup for your setup.

**Reply by Taboaco** [reply]
I’ll try that, will also try a PID change to see if it locks the yaw axis! Thanks guys
Is there any literature in how to set up a tail sitter vtol wing?

**Comment by Taboaco** [comment]
Well, if it is it’s not a locked stabilization, it’s a poor basic one, tried here, also saw that today:
https://youtu.be/voQk_KMFuJw

Em 31 de dez. de 2021, à(s) 06:47, Darren Lines ***@***.***> escreveu:

﻿

@Taboaco<https://github.com/Taboaco> Paweł and I have been chatting. Differential thrust is stabilised. If you are not getting enough effect, you will need to tune the differential thrust settings.

This could be increasing the figure in the yaw columns of the motors mixer. Or the PIDFF and rates for yaw need to be setup for your setup.

—
Reply to this email directly, view it on GitHub<https://github.com/iNavFlight/inav/discussions/7708#discussioncomment-1890943>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AUUFZAR35SY6PQNRYZKF2UDUTV34FANCNFSM5LBBMWAA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by MrD-RC** [comment]
I wasted 17 minutes watching that rotor riot video last night. That wing looks very badly set up, the guy who built it couldn’t even get it off the ground successfully. He crashed it and it yawed badly on the first flights when it didn’t even have iNav. They should just stick to being quad and hype sales people.

**Comment by Taboaco** [comment]
I also don’t like RR but he states he is using Inav and the yaw that was supposed to be stabilized is not locked, that was the point of the video, I was looking for a locked yaw like and old head lock giro of RC helis, like a quad have today, so when that guy punches the thr on takeoff he does not looses track, the wing compensates and differential thrust Kika in and automatically make it goes in a strait line, sorry for wasting your time 😔
Thanks

Em 7 de jan. de 2022, à(s) 02:51, Darren Lines ***@***.***> escreveu:

﻿

I wasted 17 minutes watching that rotor riot video last night. That wing looks very badly set up, the guy who built it couldn’t even get it off the ground successfully. He crashed it and it yawed badly on the first flights when it didn’t even have iNav. They should just stick to being quad and hype sales people.

—
Reply to this email directly, view it on GitHub<https://github.com/iNavFlight/inav/discussions/7708#discussioncomment-1922811>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AUUFZAS2QCWANQKJ6WBCLDLUUZ5ONANCNFSM5LBBMWAA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
